share/spice/dota2/dota2.js

```diff
@@ -154,15 +154,34 @@
         var returnData;
         var item = getItemFromData(api_result.itemdata, searchTarget);
         var sourceUrl;
+        var template;
         if (typeof item !== 'undefined') {
             returnData = createItemReturnObject(item, api_result);
             sourceUrl = 'https://www.dota2.com/items/';
+            template = {
+                group: 'base',
```

Comment:
you'll want to specify `item: false` -- so it doesn’t try to use an item (tile) template